### ISSUE TITLE
Properly export ffs in adios2-config when SST is disabled.

### ISSUE THREAD

**Issue by vicentebolea** [ornladios/ADIOS2#3724]
The issue happens because EVPATH dependency gets properly exported when using SST, however, even without SST we use ffs which gets exposed through EVPATH. @ax3l the workaround should be to enable SST or disable BP5.

Things to do:

- [x] Properly export ffs in adios2-config when SST is disabled.

_Originally posted by @vicentebolea in https://github.com/ornladios/ADIOS2/issues/3716#issuecomment-1656238780_

**Comment by ax3l** [comment]
Note: this issue is only with static builds:
```
cmake -S . -B build \
  -DADIOS2_USE_SST=OFF \
  -DBUILD_SHARED_LIBS=OFF
```

it appears when a dependent CMake project tries to include ADIOS2 CMake targets from `adios2-config.cmake`.

Originally seen on macOS 11.0 for cross-compiles from x86 to arm64/aarch64 with the 2.9.0 release.

**Comment by vicentebolea** [comment]
@scottwittenburg

**Comment by vicentebolea** [comment]
Solved in #3760
